### PR TITLE
Removes yellow siding tiles

### PR DESCRIPTION
**Pull request by dragomagol**
## About The Pull Request
Removes these two fellows: 
image

## Why It's Good For The Game
They're only used at Centcom, and we have decals now to do what these tiles do. 

## Changelog
:cl:
imagedel: removes the yellow siding tiles at Centcom
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
so uh, are you not going to replace the ones at centcom with the decals?

**Comment by dragomagol** [comment]
> 
> 
> so uh, are you not going to replace the ones at centcom with the decals?

image

This is what I changed it to-- this is what it would look like with the yellow lines instead:
image

I think the yellow is a little jarring, even with the old tiles.

**Comment by Fikou** [comment]
the one on the bottom looks better

File: tools/UpdatePaths/replaceplasteelturf.txt [hunk 1]
```diff
@@ -46,5 +46,3 @@
 /turf/open/floor/plasteel/white/corner : /turf/open/floor/iron/white/corner {@OLD}
 /turf/open/floor/plasteel/white/side : /turf/open/floor/iron/white/side {@OLD}
 /turf/open/floor/plasteel/white/telecomms : /turf/open/floor/iron/white/telecomms {@OLD}
-/turf/open/floor/plasteel/yellowsiding : /turf/open/floor/iron/yellowsiding {@OLD}
-/turf/open/floor/plasteel/yellowsiding/corner : /turf/open/floor/iron/yellowsiding/corner {@OLD}
```
Comment (Fikou) [inline tools/UpdatePaths/replaceplasteelturf.txt:50]: you should put this in the turfs2decals.txt script and make it convert those tiles into the siding decals

Comment (dragomagol) [reply tools/UpdatePaths/replaceplasteelturf.txt:50]: Sounds good! And I can change those back to the lines if you like. 👍

**Comment by dragomagol** [comment]
Changed back to the lines

**Review by Twaticus — APPROVED** [review]
bless u

**Review by EOBGames — APPROVED** [review]
Death to useless tile variants, so we can have more useful tile variants

**Pull request merged**